otx/core/data/anomaly_dataset_adapter.py

```diff
@@ -0,0 +1,223 @@
+"""Anomaly Classification / Detection / Segmentation Dataset Adapter."""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# pylint: disable=invalid-name, too-many-locals, no-member
+import os
+from typing import Any, List, Tuple
+
+import cv2
+import numpy as np
+from datumaro.components.dataset import Dataset as DatumaroDataset
+
+from otx.algorithms.detection.utils.mask_to_bbox import mask2bbox
+from otx.api.entities.annotation import (
+    Annotation,
+    AnnotationSceneEntity,
+    AnnotationSceneKind,
+    NullAnnotationSceneEntity,
+)
+from otx.api.entities.dataset_item import DatasetItemEntity
+from otx.api.entities.datasets import DatasetEntity
+from otx.api.entities.id import ID
+from otx.api.entities.image import Image
+from otx.api.entities.label import LabelEntity
+from otx.api.entities.label_schema import LabelSchemaEntity
+from otx.api.entities.scored_label import ScoredLabel
+from otx.api.entities.shapes.rectangle import Rectangle
+from otx.api.entities.subset import Subset
+from otx.api.utils.segmentation_utils import create_annotation_from_segmentation_map
+from otx.core.data.base_dataset_adapter import BaseDatasetAdapter
+
+
+class AnomalyBaseDatasetAdapter(BaseDatasetAdapter):
+    """BaseDataset Adpater for Anomaly tasks inherited by BaseDatasetAdapter."""
+
+    def import_dataset(
+        self,
+        train_data_roots: str,
+        val_data_roots: str = None,
+        test_data_roots: str = None,
+        unlabeled_data_roots: str = None,
+    ) -> DatumaroDataset:
+        """Import dataset by using Datumaro.import_from() method.
+
+        Args:
+            train_data_roots (str): Path for training data
+            train_ann_files (str): Path for training annotation data
```

Comment:
I don't see `train_ann_files` in arguments, should we change description accordingly?

Reply:
I'm going to delete it